### PR TITLE
add components

### PR DESCRIPTION
**Pull request by BrettKearns**

### CODE REVIEW

**Review by loganripplinger — CHANGES_REQUESTED** [review]
All of the .js files should be .jsx

File: src/features/main/Header.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export function Header() {
+  return (
+    <div>
```
Comment (loganripplinger) [inline src/features/main/Header.js:5]: React fragment here

**Review by loganripplinger — CHANGES_REQUESTED** [review]

**Comment by BrettKearns** [comment]
Changes have been made. Thank you for your review.

File: src/features/main/Header.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
```
Comment (loganripplinger) [inline src/features/main/Header.js:1]: Forgot to delete this file.

File: src/features/main/Form.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+export function Form() {
```
Comment (loganripplinger) [inline src/features/main/Form.js:1]: Forgot to delete this file.

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
```
Comment (loganripplinger) [inline src/App.js:1]: Forgot to delete this file.

**Review by loganripplinger — CHANGES_REQUESTED** [review]

**Review by loganripplinger — APPROVED** [review]

**Comment by BrettKearns** [comment]
😸

**Pull request merged**